### PR TITLE
AgentDesktop Plugin changes

### PR DESCRIPTION
**Pull request by KambagiriswamyBo-Kore**

### CODE REVIEW

File: examples/esm/chat/js/index_chat.js [hunk 1]
```diff
@@ -9,11 +9,12 @@ chatWindowInstance.installPlugin(new AgentDesktopPlugin());
 //OPTION #1
 let botOptions=chatConfig.botOptions;
 
-botOptions.JWTUrl = "PLEASE_ENTER_JWTURL_HERE";
-botOptions.userIdentity = 'PLEASE_ENTER_USER_EMAIL_ID';// Provide users email id here
-botOptions.botInfo = { name: "PLEASE_ENTER_BOT_NAME", "_id": "PLEASE_ENTER_BOT_ID" }; // bot name is case sensitive
-botOptions.clientId = "PLEASE_ENTER_CLIENT_ID";
-botOptions.clientSecret = "PLEASE_ENTER_CLIENT_SECRET";
+botOptions.koreAPIUrl = "https://bots.kore.ai/api";
```
Comment (RajasekharBa-Kore) [inline examples/esm/chat/js/index_chat.js:12]: @KambagiriswamyBo-Kore  remove configs

and validate your agent desktop changes in following two files

examples/esm/chat-with-plugins/js/index_chat.js
examples/umd/chat-with-plugins/index.html

File: package.json [hunk 2]
```diff
@@ -19,14 +19,14 @@
   "module": "dist/kore-web-sdk.esm.browser.js",
   "repository": {
     "type": "git",
-    "url": "git@github.com:Koredotcom/web-kore-sdk.git#v2/9.3.0"
+    "url": "git@github.com:KambagiriswamyBo-Kore/web-kore-sdk.git#v2/9.3.0"
```
Comment (RajasekharBa-Kore) [inline package.json:22]: @KambagiriswamyBo-Kore 

should point to koredotcom repo

File: package.json [hunk 3]
```diff
@@ -35,6 +35,7 @@
     "@types/jquery": "^3.5.6",
     "@types/pubsub-js": "^1.8.3",
     "aws-sdk": "^2.1102.0",
+    "babel-runtime": "^6.26.0",
```
Comment (RajasekharBa-Kore) [inline package.json:38]: @KambagiriswamyBo-Kore 

where are you using babelruntime ?

**Review by RajasekharBa-Kore — CHANGES_REQUESTED** [review]

File: examples/esm/chat/js/index_chat.js (hunk 1 above)
Comment (KambagiriswamyBo-Kore) [reply examples/esm/chat/js/index_chat.js]: @RajasekharBa-Kore Removed the configs

File: package.json (hunk 2 above)
Comment (KambagiriswamyBo-Kore) [reply package.json]: @RajasekharBa-Kore Updated the github url

**Pull request merged**